estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_view_tree" model="ir.ui.view">
+        <field name="name">estate.property.list</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <list string="Tests">
+                <field name="name" string="Title"/>
+                <field name="postcode"/>
+                <field name="bedrooms"/>
+                <field name="living_area" string="Living Area (sqm)"/>
+                <field name="expected_price" string="Expected Price"/>
+                <field name="selling_price" string="Selling Price"/>
+                <field name="date_availability" string="Available From"/>
+                <field name="property_type_id" string="Type"/>
+            </list>
+        </field>
+    </record>
+
+    <record id="estate_property_search" model="ir.ui.view">
+        <field name="name">estate.property.seach</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Test">
+                <field name="name" string="Title"/>
+                <field name="postcode"/>
+                <field name="expected_price" string="Expected Price"/>
+                <field name="bedrooms"/>
+                <field name="living_area" string="Living Area (sqm)"/>
+                <field name="facedes"/>
+                <field name="property_type_id" string="Type"/>
+
+                <filter name="filter_state_available" string="Available" domain="['|', ('state', '=', 'new'), ('state', '=', 'offer received')]"/>
```

Comment:
avoid to long lines. 
For long domain you can use this kind of notation:
```suggestion
                <filter name="filter_state_available" string="Available" 
                    domain="[
                        '|', 
                            ('state', '=', 'new'), 
                            ('state', '=', 'offer received')
                    ]"/>
```
Or a one liner for simple domains
```suggestion
                <filter name="filter_state_available" string="Available" 
                    domain="['|', ('state', '=', 'new'), ('state', '=', 'offer received')]"/>
```